source/template.yaml

```diff
@@ -175,6 +175,117 @@ Resources:
           ToPort: 3128
           CidrIp: 172.16.32.0/22
 
+  ProofDocumentFunction:
```

Comment:
Now that I merged https://github.com/18F/identity-idp-functions/pull/36, we don't need to touch `template.yaml` at all, we can back out these changes.